### PR TITLE
Feature/session management

### PR DESCRIPTION
**Pull request by santiagoRD**
## Summary

Create Home component with a logout button, and create tests to validate redirection after click in logout button. 

## Screenshots

Home

## Trello Card

https://trello.com/c/GrFUn3Du/83-session-management-logout

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -55,6 +60,7 @@ export const RESPONSE_STATUS = {
 };
 
 export const ROUTES = {
```
Comment (manugo-dev) [inline src/constants/index.js]: I believe you can move this to another single file /constants/routes.ts

File: src/screens/Home/index.tsx [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+import { I18N_CONFIG, HOME_FIELDS, ROUTES } from 'constants/index';
+import logo from 'assets/wLogo.png';
 
-import logo from './assets/logo.svg';
 import styles from './styles.module.scss';
 
 function Home() {
+  const history = useHistory();
+  const handleLogout = () => {
+    localStorage.removeItem('token');
```
Comment (manugo-dev) [inline src/screens/Home/index.tsx]: Move this function inside utility file. `utils/session.ts`

File: src/screens/Login/index.tsx [hunk 3]
```diff
@@ -25,8 +32,9 @@ function Login() {
     request: UserService.loginUser,
     withPostSuccess: (data, headers) => {
       setSuccess(`${I18N_CONFIG.key.login}:${SUCCESS_MESSAGES.userLogged}`);
-      // eslint-disable-next-line no-console
-      console.log(data, headers);
+      const token = headers?.accessToken || '';
+      localStorage.setItem('token', token);
```
Comment (manugo-dev) [inline src/screens/Login/index.tsx]: same for it.

**Review by manugo-dev — CHANGES_REQUESTED** [review]

File: public/index.html [hunk 4]
```diff
@@ -5,15 +5,17 @@
...
-      content="Web site created using create-react-app"
-    />
+    <meta name="description" content="Web site created using create-react-app" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
```
Comment (CrisVega08) [inline public/index.html]: Import font in scss file

File: src/config/api.js [hunk 5]
```diff
@@ -21,6 +21,7 @@ const api = create({
 api.addResponseTransform(response => {
```
Comment (CrisVega08) [inline src/config/api.js]:
```suggestion
api.addResponseTransform(({data, headers}) => {
```

File: src/routes/index.tsx [hunk 6]
```diff
@@ -3,13 +3,15 @@ import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
...
 import Login from 'screens/Login';
 import SignUp from 'screens/SignUp';
+import Home from 'screens/Home';
 import { ROUTES } from 'constants/index';
 
 function Routes() {
   return (
     <Router>
       <Switch>
         <Route path={ROUTES.signUp} component={SignUp} />
```
Comment (CrisVega08) [inline src/routes/index.tsx]: You can improve your const Routes adding path, component and other properties. And here you can loop it for create dynamic routes
```
Routes.map(({component, path , exact}) => <Route path={path} component={component} />)

```

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Review by manugo-dev — APPROVED** [review]

**Pull request merged**